tests/functional/spec/save_sign_out.spec.js

```diff
@@ -51,6 +51,7 @@ describe("Save sign out / Exit", () => {
 
   it("Given a I have started a social questionnaire, when I select save and sign out, then I am redirected to the signed out page and the correct access code link is shown", async () => {
     await browser.openQuestionnaire("test_theme_social.json", { version: "v2", theme: "social" });
+    await expect(await $(SubmitPage.saveSignOut()).getAttribute("href")).to.contain("/sign-out?internal_redirect=True");
```

Comment:
This isn't what I was getting at, the redirect URL is what you need to assert. This is implicitly asserted by line 56.
You need to assert it is the todo url for non social basically and log out url for the social survey.